<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShaderViewClass</class>
 <widget class="QWidget" name="ShaderViewClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="ShaderView" name="shaderView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Shader: name</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="vertexLayout">
        <item>
         <widget class="QLabel" name="vertexLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Vertex File:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="FileHandler" name="vertexFile">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="vertexBtn">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="fragmentLayout">
        <item>
         <widget class="QLabel" name="fragmentLabel">
          <property name="text">
           <string>Fragment File:  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="FileHandler" name="fragmentFile">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fragmentBtn">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="macrosies">
        <property name="title">
         <string>Macrosies:</string>
        </property>
        <layout class="QVBoxLayout" name="macrosiesLayout">
         <item>
          <layout class="QHBoxLayout" name="addElementLayout">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="addMacroBtn">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>31</height>
              </size>
             </property>
             <property name="text">
              <string>add</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="properties">
        <property name="title">
         <string>Properties:</string>
        </property>
        <layout class="QVBoxLayout" name="propertiesLayout">
         <item>
          <layout class="QHBoxLayout" name="addElementLayout_2">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="addPropertyBtn">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>31</height>
              </size>
             </property>
             <property name="text">
              <string>add</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="buttonsLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="revertBtn">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Revert</string>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="applyBtn">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ShaderView</class>
   <extends>QGroupBox</extends>
   <header>PropertyViews/ShaderView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FileHandler</class>
   <extends>QLineEdit</extends>
   <header>PropertyViews/FileHandler.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
